dag/set_test.go

```diff
@@ -119,3 +119,13 @@ func TestSetCopy(t *testing.T) {
 	}
 
 }
+
+func BenchmarkSetIntersectionLargeToSmall(b *testing.B) {
+	var small, large Set
```

Comment:
I'm not  certain how you were executing the benchmark, but this should panic as the zero value of `Set` is not usable. These need to be initialized with `make` or a literal.

Reply:
right! I'm working on a bit of franken-version here (still trying to get up to speed at github)...

Clearly the benchmark alisdair@ did is definitely better. Should I just revert the change in the test then and pick his version?

Reply:
oops, my review collided with @alisdair's. Yes, using the benchmarks he posted would also be fine. Thanks!